.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "in-review"]
+assignees:
+  - glharper
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 1.25.0
+        - 1.26.0
+        - 1.27.0
+        - 1.28.0
+        - 1.29.0
+        - 1.30.0
+        - 1.30.1 (Default)
+        - 1.31.0 (Edge)
+        - Older 
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browser/platform are you seeing the problem on?
+      multiple: true
+      options:
+        - Node
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - React
+        - React Native
+        - Other
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Is there a docs link we could add to show users how to turn on logging and collect the output?
We have a link like that in the carbon samples repo bug template.